.github/workflows/Java_CI.yaml

```diff
@@ -0,0 +1,16 @@
+name: Java CI
+
+on:
+  push:
+    branches: [ Feature/wrims-core ]
```

Comment:
I would revise this to all pull requests into "wrims-devops" as well as the "wrims-devops" branch itself: for example: https://github.com/opendcs/rest_api/blob/main/.github/workflows/default.yml#L5 replace "main" with "wrims-devops"

Reply:
Concur. Changed